### PR TITLE
Announcement and download links for 0.20.0

### PR DESCRIPTION
**Pull request by selaux**

### CODE REVIEW

File: _posts/_future-release-0.20.0.md [hunk 1]
```diff
@@ -0,0 +1,183 @@
+---
+layout: post
+title: Release 0.20.0
+---
+
+The Jagged Alliance 2 Stracciatella team is proud to present a new release showcasing about half a year of work. It includes several new features, plenty of bugfixes compared to previous versions and the original game, clean-ups of the old code base, and most notably the first official release of the Android version and support for touch controls.
+
+Through 490 commits at least 38 bugs were fixed, 8 of which are known to affect vanilla.
```
Comment (lynxlynxlynx) [inline _posts/_future-release-0.20.0.md]: These stats don't match the changelog, where more vanilla bugs are listed.

**Comment by lynxlynxlynx** [comment]
Other than the todos, looks good!

File: _posts/_future-release-0.20.0.md (hunk 1 above)
Comment (selaux) [reply _posts/_future-release-0.20.0.md]: Fixed. Previously I used the bug tracker, now I used the lines below.

**Comment by lynxlynxlynx** [comment]
I just noticed the gifs are huge, almost 30M together. Please recode them to mp4 and use a video element, so we get compression and they won't bog down the page.

**Pull request merged**